### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by micronaldkelly** [micronaldkelly/codelab-js-functions-strings-variables#1]
Hey @kieraw, can you take a look at this? It's [hosted here](hapless-toys.surge.sh) and meets the following [project](http://lansingcodelab.com/lessons/js-functions-strings-variables/1) criteria:
- [x] an HTML profile with a name, favorite color, and at least one other attribute
- [x] the containing element for each attribute is blank initially
- [x] prompts are used to collect text values for each attribute
- [x] each attribute is given an `id` (e.g. "name", "favorite-color"), which is used to target that element and set its `innerHTML` to the value provided by the user

**Comment by ghost** [comment]
@micronaldkelly getting a "page not found" error when I try to access your site here: http://hapless-toys.surge.sh/ Please resolve and let me know when you do!
--Kiera

**Comment by micronaldkelly** [comment]
I am going to copy and paste my code because when i go into js lint it tells me i'm missing something that is there.  All help appreciated!

<!doctype html>
<html>
<body>

<p id="name"</p>

<p id="color"</p>

<p id="food"</p>

<script>
var name=prompt("Please enter your name")
//("Welcome " + name + "! I hope you are ready to embark on a magical journey!")
var color = prompt("Please enter favorite color")
//("Why is " + color + " your favorite color if you don't even wear it?")
var food = prompt("Please enter your favorite food")
//("Eat dat " + food)
document.getElementById('name').innerHTML = "Welcome " + name + "! I hope you are ready to embark on a magical journey!";
document.getElementById('color').innerHTML = "Why is " + color + " your favorite color if you dont even wear it?";
document.getElementById('food').innerHTML = "Eat dat " + food;
</script>

</body>
</html>

**Comment by ghost** [comment]
@micronaldkelly it looks like your code didn't copy correctly!
--Kiera

**Comment by micronaldkelly** [comment]
<!doctype html>
<html>
<body>

<p id="name"</p>

<p id="color"</p>

<p id="food"</p>

<script>
var name=prompt("Please enter your name")
//("Welcome " + name + "! I hope you are ready to embark on a magical journey!")
var color = prompt("Please enter favorite color")
//("Why is " + color + " your favorite color if you don't even wear it?")
var food = prompt("Please enter your favorite food")
//("Eat dat " + food)
document.getElementById('name').innerHTML = "Welcome " + name + "! I hope you are ready to embark on a magical journey!";
document.getElementById('color').innerHTML = "Why is " + color + " your favorite color if you dont even wear it?";
document.getElementById('food').innerHTML = "Eat dat " + food;
</script>

</body>
</html>

**Comment by micronaldkelly** [comment]
damnit
 how do i send my code

**Comment by ghost** [comment]
@micronaldkelly I can just view it in your repo, don't worry! Do you have an updated URL for the site?
--Kiera

**Comment by micronaldkelly** [comment]
acceptable-bottle.surge.sh

**Comment by ghost** [comment]
@micronaldkelly make sure to put semicolons after your lines of javascript, like so:
var name=prompt("Please enter your name");
Also, that URL is broken as well. 
--Kiera

**Comment by micronaldkelly** [comment]
I just put semicolons after the lines of js and the URL is still broken.  Can you look at my updated code that I just saved?

**Comment by ghost** [comment]
@micronaldkelly The code isn't returning any JavaScript Lint errors. Perhaps the surge url isn't being generated in the directory where your project is, I am unsure.
--Kiera

**Comment by micronaldkelly** [comment]
Yeah I guess I'll leave it until tomorrow because I keep going through the motions in terminal and it's not giving me a URL that works.  I suppose I will have to do without you tomorrow!

**Comment by ghost** [comment]
@micronaldkelly that's very odd. Hopefully tomorrow will offer a solution. Good luck and let me know when it is resolved!
--Kiera

**Comment by micronaldkelly** [comment]
will do!

**Comment by micronaldkelly** [comment]
short-plate.surge.sh

Couldn't leave you hanging!!!!

**Comment by ghost** [comment]
@micronaldkelly looks great! :shipit: 
--Kiera
